asistente_ladm_col/gui/reports/reports.py

```diff
@@ -158,30 +156,8 @@ def get_tmp_filename(self, basename, extension='gpkg'):
     def generate_report(self, db, report_type):
         # Check if mapfish and Jasper are installed, otherwise show where to
         # download them from and return
-        base_path = os.path.join(DEPENDENCIES_BASE_PATH, DEPENDENCY_REPORTS_DIR_NAME)
-        bin_path = os.path.join(base_path, 'bin')
-        if not os.path.exists(bin_path):
-            self.logger.message_with_button_download_report_dependency_emitted.emit(
-                QCoreApplication.translate("ReportGenerator",
-                   "The dependency library to generate reports is not installed. Click on the button to download and install it."))
-            return
-
-        # Check version
```

Comment:
Where do we check the report version after this PR?

Reply:
Now this validation is in report dependency. https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/6c5f2f2810b1b4ec674cf2512530f6ec9902faed/asistente_ladm_col/lib/dependency/report_dependency.py#L83

Reply:
:+1: